src/actions/general.js

```diff
@@ -115,7 +123,8 @@ export function getLicenseConfig() {
     );
 }
 
-export function logClientError(message, level = 'ERROR') {
+type logLevel = 'ERROR' | 'WARNING' | 'INFO' // I guess?
```

Comment:
this type, probably, will be used in `src/client/client4.js` file too, so I would move it somewhere in `types` directory

Reply:
Time to make a client4 types file!
(and immediately change some work I did in the other PR 🙈)

Reply:
Client4.js will be a big amount of work, If you want, go for it, and if you need any kind of help, let me know and I'll try to help as much as possible. :)